src/main/java/com/oltpbenchmark/api/Procedure.java

```diff
@@ -156,6 +158,24 @@ protected final void loadSQLDialect(StatementDialects dialects) {
         }
     }
 
+    /**
+     * Enable monitoring for this procedure by adding a monitoring prefixes.
+     */
+    protected final void enabledAdvancedMonitoring() {
+        for (String stmtName : this.getStatements().keySet()) {
+            SQLStmt stmt = this.name_stmt_xref.get(stmtName);
+            if (LOG.isDebugEnabled()) {
+                LOG.debug(String.format("Enabling advanced monitoring for query %s",
```

Comment:
```suggestion
                LOG.debug(String.format("Enabling advanced monitoring for query %s",
```

@timveil pointed out that the easier way to do this now is with "parameterized logging"
https://www.slf4j.org/faq.html#logging_performance
So then you can also drop the `if (LOG.isDebugEnabled())` since the string parameterization parsing overhead will only take effect when debug is actually enabled.
I think there are cases where the arguments passed to the string format might still be expensive so the `if (LOG.isDebugEnabled())` check might still make sense then, but for others it'll be more readable to use the parameterized format and drop the check.